TurkishFilter/sections/specific.txt

```diff
@@ -45,6 +45,9 @@ merlininkazani.com##.adsbygoogle
 ! https://github.com/AdguardTeam/AdguardFilters/issues/19218
 !+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)
 dizired.com##.adv-sidebar
+! https://github.com/AdguardTeam/AdguardFilters/issues/41925
+!+ PLATFORM(ios, ext_android_cb)
+dizibox.pw##.container > a[rel="nofollow"][target="_blank"] > img
```

Comment:
not a bad rule. Why not just add to mobile filter?

Reply:
on android device I wasn't able to reproduce this ad, that's why I decided to limit rule, but in other case it might be to add in mobile filter.